README.md

```diff
@@ -1,2 +1,89 @@
-# aws-nextflow-workflows
-AWS CloudFormation templates for deploying Nextflow infrastructure
+# Nextflow Infrastructure
+
+The AWS infrastructure for hosting a [private instance](https://tower.sagebionetworks.org) of [Nextflow Tower](https://tower.nf/) and executing [Nextflow workflows](https://nextflow.io/) is defined in this repository and deployed using [CloudFormation](https://aws.amazon.com/cloudformation/) via [Sceptre](https://sceptre.cloudreach.com/).
+
+➡️ **Nextflow Tower for Sage Bionetworks:** https://tower.sagebionetworks.org
+
+## Getting Started
+
+### Prospective Tower Users
+
+Follow the [Tower User Onboarding](#tower-user-onboarding) instructions below. Access is currently restricted to Sage Bionetworks staff.
+
+### Prospective Contributors
+
+Read through the [contribution guidelines](CONTRIBUTING.md) for more information. Contributions are welcome from anyone!
+
+## Tower User Onboarding
+
+1. Determine what is known as the stack name by concatenating the project name with the suffix `-project` (_e.g._ `imcore-project`, `amp-ad-project`, `commonmind-project`).
+
+   **Important:** Anytime that `<stack_name>` appears below with the angle brackets, replace the placeholder with the actual stack name, omitting any angle brackets.
+
+2. Install [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) and [Docker](https://docs.docker.com/engine/install/) on your local computer or on a server available to you.
+
+3. [Create](https://sagebionetworks.jira.com/jira/software/c/projects/IT/issues/) an IT JIRA ticket requesting membership to the following JumpCloud groups for anyone who needs read-only or read/write access to this project:
+
+   - `aws-sandbox-developers`
+   - `aws-workflow-nextflow-tower-viewer`
+
+   To confirm whether you're already a member of these JumpCloud groups, you can expand the AWS Account list on [this page](https://d-906769aa66.awsapps.com/start#/) (after logging in with JumpCloud) and check if you have `Developer` listed under `org-sagebase-sandbox` and `TowerViewer` under `workflows-nextflow-dev` and `workflows-nextflow-prod`.
+
+   ![AWS SSO Screenshot](assets/img/aws_sso.png)
+
+4. Open a pull request on this repository with the following changes:
+
+   - Duplicate [`config/prod/example-project.yaml`](config/prod/example-project.yaml) as `<stack_name>.yaml` in the `prod/` subdirectory and then follow the numbered steps listed in the file. Note that some steps are required whereas others are optional.
+   - Add any missing emails to the [`tower-config/tower.yaml`](tower-config/tower.yaml) file
```

Comment:
What does it mean to have read/write access vs read?  Is it that people with read permissions are able to view the workflow run results, but not able to start workflow runs?

Reply:
So far, read/write vs read-only only determines what kind of access someone has to the encrypted project bucket. I added a note to clarify. 

That said, Tess and I are working on automating the setup in Tower, and this will likely include the bootstrapping of a Tower workspace within a Sage organization and we'll map these levels of access (read/write vs read-only) to roles in Tower (see screenshot below for available roles). 

![image](https://user-images.githubusercontent.com/740725/132439914-ee4ce28c-13d3-4c39-bbf0-4a6fd63d3695.png)